hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-shuffle/src/main/java/org/apache/hadoop/mapred/ShuffleHandler.java

```diff
@@ -904,11 +986,11 @@ public void setPort(int port) {
     }
 
     @Override
-    public void channelOpen(ChannelHandlerContext ctx, ChannelStateEvent evt) 
+    public void channelActive(ChannelHandlerContext ctx)
         throws Exception {
-      super.channelOpen(ctx, evt);
-
-      if ((maxShuffleConnections > 0) && (accepted.size() >= maxShuffleConnections)) {
+      LOG.debug("channelActive");
```

Comment:
This is a really vague message, we don't know what channel, and under a heavy load we would only see a lot of "channelActive" messages, without context. Especially the whole netty is asynchronous (and multithreaded) so it would be quite impossible to identify this log. Isn't there a way to add a channel ID or something? In this case we could use that for all messages generated by the channel

Reply:
Very much making sense. Fixed a lot of logging messages in the class.